admin_guide/manage_users.adoc

```diff
@@ -9,127 +9,103 @@
 
 toc::[]
 
-== Adding Users
+[[managing-users-overview]]
```

Comment:
In the metadata at the top of the topic, it needs a `:prewrap!:` so that we get horizontal scroll bars in code output. See:

https://raw.githubusercontent.com/openshift/openshift-docs/master/contributing_to_docs/templates/topic_template.adoc